_includes/releases/v22.1/v22.1.0-beta.3.md

```diff
@@ -0,0 +1,78 @@
+## v22.1.0-beta.3
+
+Release Date: April 18, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-3-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Unified the syntax for defining the behavior of initial scans on [changefeeds](../v22.1/create-changefeed.html) by extending the `initial_scan` option to accept three possible values: `'yes|no|only'`. [#79471][#79471]
+- Changefeed messages for tables with multiple [column families](../v22.1/column-families.html) using the `split_column_families` option or the [`CREATE CHANGEFEED for TABLE bar FAMILY f_1`](../v22.1/create-changefeed.html) syntax will append the family name to the table name in the default topic. For example, `CREATE CHANGEFEED FOR TABLE foo, TABLE bar FAMILY f_1` will emit to the topics `foo` and `bar.f_1`.  In cloud storage sinks such as s3, the filename will include the family separated by a `+` (e.g., `bar+f_1`). A changefeed created with the `split_column_families` option will create new topics if new families are added to a watched table that already had multiple column families. [#79448][#79448]
```

Comment:
Yes, this looks good, but I actually think this could be shortened since the docs for the feature were merged last week? 

Changefeeds can now target tables with [more than one column family](use-changefeeds.html#changefeeds-on-tables-with-column-families) using either the [`split_column_families` option](create-changefeeds.html#split-column-families) or the `FAMILY` keyword . Changefeeds will emit individual messages per column family on a table.

Either way it looks good to me, here are some relevant links:
**The column family work**: use-changefeeds.html#changefeeds-on-tables-with-column-families, provides a description of the working & gotchas 
**example**: create-a-changefeed-on-a-table-with-column-families. 
**option anchor link**: create-changefeeds.html#split-column-families